tests/wpunit/Tribe/Date_UtilsTest.php

```diff
@@ -252,6 +252,25 @@ public function test_reformat( $input, $format ) {
 		$this->assertEquals( $date->format( 'U' ), Date_Utils::reformat( $date->format( 'U' ), 'U' ) );
 	}
 
+	/**
+	 * Test reformat w/ timezone.
```

Comment:
What about adding invalid timestamps to the @dataProvider reformat_inputs to verify it can handle them without breaking errors?